docs/tables/aws_amplify_app.md

```diff
@@ -0,0 +1,110 @@
+# Table: aws_amplify_app
+
+Represents the different branches of a repository for building, deploying, and hosting an Amplify app.
```

Comment:
@omerosaienni Can we add a better description for the table here? 
For reference - https://aws.amazon.com/amplify/

Reply:
That is the actual description of what the App is and not what Amplify is. The page you sent me to is to explain Amplify and I am not sure that is what we want to describe here. 

Here is where I got the description: 
https://docs.aws.amazon.com/amplify/latest/APIReference/API_App.html

Reply:
What I have done here is filled out the sentence and made it flow better. 

I am simply describing what the `app` table resource represents in this description.

Reply:
In the table description, we generally provide information about the service and resource. In this case, it’s a little bit hard to frame it. I think we can keep the earlier one with a little bit of modification.